File: Java/03. Code Related to Arrays/71. Balanced Parenthesis Problem.java
import java.util.*;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.print("Enter the string of brackets: ");
        String str = userInput.next();

        while ( str.length() != 0 ) {
            int original = str.length();

            str = str.replace("(","");
            str = str.replace("[","");
            str = str.replace("{","");
            str = str.replace(")","");
            str = str.replace("]","");
            str = str.replace("}","");

            if( (str.length()) == original ) {
                break;
            }
        }

        if ( str.length() == 0) {
            System.out.println("The Entered String has Balanced brackets");
        }
        else {
            System.out.println("The entered string does not have balanced brackets");
        }
        userInput.close();
    }
}

File: Python/03. Code Related to Arrays/71. Balanced Parenthesis Problem.py
string = input("Enter the brackets:")

while( len(string) != 0 ):
        original = len(string)

        string = string.replace('(','')
        string = string.replace('[','')
        string = string.replace('{','')
        string = string.replace(')','')
        string = string.replace(']','')
        string = string.replace('}','')

        if( len(string) == original ):
             break

if ( len(string) == 0):
   print("The Entered String has Balanced brackets")
   
else:
    print("The entered string does not have balanced brackets")